Makefile

```diff
@@ -7,7 +7,7 @@ endif
 
 PROJECT := github.com/juju/juju
 PROJECT_DIR := $(shell go list -e -f '{{.Dir}}' $(PROJECT))
-PROJECT_PACKAGES := $(shell go list $(PROJECT)/... | grep -v /vendor/)
+PROJECT_PACKAGES := $(shell go list $(PROJECT)/... | grep -v /vendor/ | grep -v /acceptancetests/)
```

Comment:
I'm curious why this was needed. Did we end up with an acceptance test in go that is causing problems?

Reply:
We end up attempting to install everything in acceptance tests, which we shouldn't do to begin with. Also CI also has an [issue](https://jenkins.juju.canonical.com/job/github-make-check-juju/90/console) with the following error:

```console
15:07:41 go install github.com/juju/juju/acceptancetests/repository/trusty/fill-logs/actions: build output "/home/ubuntu/go/bin/actions" already exists and is not an object file
15:07:56 Makefile:82: recipe for target 'go-install' failed
```